File: src/minimuse/community/fortran_example/interface.f90
FUNCTION echo_int(input, output)
    INTEGER echo
    INTEGER echo_int
    INTEGER input, output
    output = echo(input)
    echo_int = 0
END FUNCTION

FUNCTION add_one(output)
    use fortran_minilib
    INTEGER output
    call add1(output)
    add_one = 0
END FUNCTION

File: src/minimuse/community/c_example/interface.cc
#include "minilib.h"

extern int echo(int input);

/*
 * Interface code
 */
 
int echo_int(int input, int * output){
    *output = echo(input);
    return 0;
}

int add_one(int * output){
    add1(output);
    return 0;
}

